testsuite/SystemInfo/proc-mdstat.cc

```diff
@@ -69,9 +69,9 @@ BOOST_AUTO_TEST_CASE(parse2)
     };
 
     vector<string> output = {
-	"data[md125] -> md_type:raid1 super:external:/md127/0 size_k:4194304 readonly devices:</dev/sdb /dev/sda> has_container container_name:md127 container_member:0",
-	"data[md126] -> md_type:raid0 super:external:/md127/1 chunk_k:128 size_k:8378790 devices:</dev/sdb /dev/sda> has_container container_name:md127 container_member:1",
-	"data[md127] -> md_type:unknown super:external:imsm size_k:5104 readonly inactive devices:<> spares:</dev/sda /dev/sdb> is_container"
+	"data[md125] -> md-level:RAID1 super:external:/md127/0 size_k:4194304 readonly devices:</dev/sdb /dev/sda> has_container container_name:md127 container_member:0",
```

Comment:
Please unify `size_k` and `size-k`, both seem to be in use now, is that for some purpose?